### PR TITLE
[13.0][ADD] edi_voxel_sale_order_import: New module

### PR DESCRIPTION
**Pull request by ernestotejeda**
This module allows you to import sales orders from Voxel. Imports are queued in jobs running in background.

Depends on:
- [x] https://github.com/OCA/edi/pull/237

cc @Tecnativa TT22083

### CODE REVIEW

**Review by joao-p-marques — APPROVED** [review]

File: edi_voxel_sale_order_import/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+                    date_format += "%H:%M:%S"
+                vals.update(
+                    commitment_date=datetime.strptime(begin_date, date_format),
+                    date_order=datetime.strptime(begin_date, date_format),
+                )
+            # add validity_date
+            end_date = general_data.get("EndDate")
+            if end_date:
+                # add validity_date
+                vals["validity_date"] = datetime.strptime(end_date, "%Y-%m-%d").date()
```
Comment (victoralmau) [inline edi_voxel_sale_order_import/models/sale_order.py]:
```suggestion
                vals.update(validity_date=datetime.strptime(end_date, "%Y-%m-%d").date())
```

**Review by victoralmau — COMMENTED** [review]
Good job!

**Comment by ernestotejeda** [comment]
@victoralmau changes done. Thanks.

File: edi_voxel_sale_order_import/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+            domains.append([("name", "=", data["name"])])
+        if data.get("email"):
+            domains.append([("email", "=", data["email"])])
+        if data.get("zip"):
+            domains.append([("zip", "=", data["zip"])])
+        elif data.get("city"):
+            domains.append([("city", "=", data["city"])])
+        partner = self.env["res.partner"].search(expression.AND(domains))
+        if len(partner) == 1:
+            return  # return the unique partner matching
```
Comment (FernandoRomera) [inline edi_voxel_sale_order_import/models/sale_order.py]: return partner # return the unique partner matching

Comment (ernestotejeda) [reply edi_voxel_sale_order_import/models/sale_order.py]: Done

**Comment by pedrobaeza** [comment]
Merging after the merge of the main module:

/ocabot merge nobump

**Pull request merged**